script/Ciel/Day 11/11_01_CIEL11_2_1.txt

```diff
@@ -298,17 +315,18 @@ I'll have to leave it for last, or perhaps―――
 [sha:02a1baee1653a3178a9b33eb8676767ac87b22d0]{
 -- Page 114, Offset 29385.
 -- まだ誰も変化に気付いていない<最|い><初|ま>のうちに、不意を突いて済ませるしかない。
-I have no choice but to ambush it and take it out right now, while no one has yet to notice my transformation.
+I have no choice but to ambush it and take it out right now, while everyone has yet to notice my transformation.
 }
 [sha:c67e09c30a630a2681b97fa8b95a808034c27357]{
 -- Page 115, Offset 29386.
 -- 「上出来だ。一縷の望みにかけた甲斐があった。
-"Good shit. Paid off to have bet on this one sliver of hope.
+"Good shit. Looks like my bet on this one sliver of hope paid off.
 }
 [sha:ae08e25b13f2a67772e5991fd40ef4574371cc51]{
 -- Page 115, Offset 29387.
 -- テメェは電波情報の<癌|がん>だ。回線が繋がっている肉体ならたやすく安全な方に転移できる。
-You're a fuckin' cancer of a signal transmission. As long as your lines are connected, you can easily and safely transfer to another body.
+You're a fuckin' cancer of a signal transmission.
+ So long as your lines are connected, you got an easy 'n risk free way to hop bodies.
```

Comment:
"cancer of a signal transmission" sounds a bit wrong, when he doesn't mean it like an insult, but more as an actual likeness.
>You're a fuckin' signal transmitted cancer.